### PR TITLE
examples added

### PR DESCRIPTION
**Pull request by picibucor**
these examples explains how id's work.

### CODE REVIEW

**Comment by dessant** [comment]
Thanks for the contribution. :+1: It will be reviewed soon after 1.9.0 is released.
As a general recommendation, don't use spaces in folder or file names, you could replace them with underscores.

**Comment by inclement** [comment]
The content looks good to me, though it will indeed be necessary to use filenames without spaces. I think an appropriate directory might be examples/kv/ids - what do others think?

**Comment by dessant** [comment]
@inclement +1.

**Comment by picibucor** [comment]
I got stuck:
fail

File: examples/kv/ids/kv_and_py/kv_and_py.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+kivy.require('1.8.0')
+
+
+class RootWidget(BoxLayout):
+
+    def first_function(self, status):
+        # print out the given parameter
+        print(status)
+        # check the status of the switch by referring on the id
+        if self.ids['my_switch'].active is True:
```
Comment (dessant) [inline examples/kv/ids/kv_and_py/kv_and_py.py]: You could do `self.ids.my_switch.active` instead, so it conforms with the rest of the examples and docs.

**Comment by dessant** [comment]
Ready for merging after that small fix. :+1:

**Comment by picibucor** [comment]
@dessant : done

**Comment by dessant** [comment]
Thanks a lot. :cake:

**Pull request merged**